Libraries/CO.CDP.Forms.WebApiClient.Tests/FormsClientIntegrationTest.cs

```diff
@@ -0,0 +1,25 @@
+using CO.CDP.TestKit.Mvc;
+using FluentAssertions;
+using Xunit.Abstractions;
+
+namespace CO.CDP.Forms.WebApiClient.Tests;
+
+public class FormsClientIntegrationTest(ITestOutputHelper testOutputHelper)
+{
+    private readonly TestWebApplicationFactory<Program> _factory = new(builder =>
+    {
+        builder.ConfigureLogging(testOutputHelper);
+    });
+
+    [Fact]
+    public async Task ItTalksToTheFormsApi()
+    {
+        IFormsClient client = new FormsClient("https://localhost", _factory.CreateClient());
+        var formId = Guid.Parse("a38b44a4-7a13-409d-912e-b88b9958e54a");
+        var sectionId = Guid.Parse("3b7a1483-199c-431f-a230-e8a5e572dec4");
+
+        var formSection = await client.GetFormSectionQuestionsAsync(formId, sectionId);
+
+        formSection.Should().NotBeNull();
```

Comment:
@glgoaco this test might start failing when we replace the stub with an actual service. I'm happy to update it later.

The goal of this test is just to make sure that the client can talk to the service. For that, only one call (any) is necessary. We usually go with the simplest.